ynput/OpenPype
# Resolve: loader is opening even if no timeline in project

## Changelog Description
Loader is opening now even no timeline is available in a project.

## Testing notes:
1. open Resolve in any new empty project
2. Click at Loader at OpenPype menu
3. notice that new timeline is created and loader is offering loadable items.
